modules/update-availability-faq.adoc

```diff
@@ -0,0 +1,67 @@
+
+[id="channel-safety_{context}"]
+*Is a release on the `stable` channel safer or more supported than a release on the `fast` channel?*
+
+* If a regression is identified for a release on a `fast` channel, it will be resolved and managed to the same extent as if that regression was identified for a release on the `stable` channel.
+
+* The only difference between releases on the `fast` and `stable` channels is that a release only appears on the `stable` channel after it has been on the `fast` channel for some time, which provides more time for new update risks to be discovered.
+
+** This delay is used to further test and gather data about a release that is on the `fast` channel, but otherwise there is no difference between releases on each of the channels.
+
+[id="supported-updates_{context}"]
+*What does it mean if an update is supported but not recommended?*
+
+* Red Hat continuously evaluates data from multiple sources to determine whether updates from one version to another lead to issues.
+If an issue is identified, an update path may no longer be recommended to users.
+However, even if the update path is not recommended, customers are still supported if they perform the update.
+
+* Red Hat does not block users from updating to a certain version.
+Red Hat may declare conditional update risks, which may or may not apply to a particular cluster.
+
+** Declared risks provide cluster administrators more context about a supported update.
+Cluster administrators can still accept the risk and update to that particular target version.
+This update is always supported despite not being recommended in the context of the conditional risk.
+
+[id="removed-recommendation_{context}"]
+*What if I see that an update to a particular release is no longer recommended?*
+
+* If Red Hat removes update recommendations from any supported release due to a regression, a superseding update recommendation will be provided to a future version that corrects the regression.
+There may be a delay while the defect is corrected, tested, and promoted to your selected channel.
+
+[id="z-stream-release-cadence_{context}"]
+*How long until the next z-stream release is made available on the fast and stable channels?*
+
+* While the specific cadence can vary based on a number of factors, new z-stream releases for the latest minor version are typically made available about every week. Older minor versions, which have become more stable over time, may take much longer for new z-stream releases to be made available.
++
+[IMPORTANT]
+====
+These are only estimates based on past data about z-stream releases. Red Hat reserves the right to change the release frequency as needed. Any number of issues could cause irregularities and delays in this release cadence.
+====
+
+* Once a z-stream release is made available, it also appears in the `fast` channel for that minor version. After a delay, the z-stream release may then appear in that minor version's `stable` channel.
```

Comment:
Looks like it's kind of duplicate with the answer of "What are the differences between each of the update channels". If we want to keep it, is it possible to replace `is made available` with `is published` or `is shipped alive` or something such?

Reply:
It is a duplicate, but I think it's worth keeping in case someone doesn't read that question but would still find that detail helpful for this question.

I can change "is made available" to say "is published" instead, thank you.